ern-local-cli/src/commands/cauldron/add/publisher.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import {
+  Utils,
+  promptUtils
+} from 'ern-util'
+import {
+  cauldron
+} from 'ern-core'
+
+exports.command = 'publisher'
+exports.desc = 'Add a publisher url to publish the generated container/archive.'
+
+exports.builder = (yargs: any) => {
+  return yargs
+    .option('mavenUrl', {
+      type: 'string',
+      describe: 'Maven url to publish the generated aar file for Android container.'
+    })
+    .option('githubUrl', {
+      type: 'string',
+      describe: 'Github url to publish the generated framework project(if ios) or library project (if android)'
+    })
+    .option('platform', {
+      type: 'string',
+      describe: 'Platform(ios or android) for which the publisher needs to be added.'
+    })
+}
+
+exports.handler = async function ({
+                                    mavenUrl,
+                                    githubUrl,
+                                    platform
+                                  }: {
+  mavenUrl?: string,
+  githubUrl?: string,
+  platform?: string
+}) {
+  try {
+    let url = ''
+    if (mavenUrl) {
+      url = mavenUrl
+    } else if (githubUrl) {
+      url = githubUrl
+    } else {
+      throw new Error(`Please provide a publisher option (publisher --mavenUrl or --gitHubUrl)`)
+    }
+
+    log.debug(`Adding ${mavenUrl ? 'maven' : 'gitHub'}publisher: ${url}`)
+
+    if (!platform || (platform !== 'ios' && platform !== 'android')) {
```

Comment:
Typically `ios` is written as `iOS` , comparing platform.toLowerCase() could be a consideration? that would leave the users options to remain case agnostic , so for android.